default/index.js

```diff
@@ -46,5 +46,6 @@ module.exports = {
   getPowerState: () => Promise.resolve({}),
   hasSystemFeature: () => Promise.resolve(false),
   getSystemAvailableFeatures: () => Promise.resolve([]),
-  getCameraPresence: () => Promise.resolve(false)
+  getCameraPresence: () => Promise.resolve(false),
+  hasLocationServicesEnabled: () => Promise.resolve(false)
```

Comment:
this is very minor - but I'm working on react-native-community lint standards with #663 (not merged yet) and it should have a trailing comma